limbo/slack.py

```diff
@@ -98,17 +71,17 @@ def process_changes(self, data):
         if "type" in data.keys():
             if data["type"] in ('channel_created', 'group_joined'):
                 channel = data["channel"]
-                self.attach_channel(channel["name"], channel["id"], [])
+                self.channels[channel["id"]] = Channel(channel["id"], channel["name"])
             if data["type"] == "im_created":
                 channel = data["channel"]
-                self.attach_channel(channel["user"], channel["id"], [])
+                self.channels[channel["id"]] = Channel(channel["id"], channel["name"])
             elif data["type"] == "team_join":
                 user = data["user"]
-                self.parse_user_data([user])
+                self.parse_users([user])
             pass
```

Comment:
remove `pass`

Reply:
yeesh yeah. This code is still mostly straight from the slackhq python library, which is less than stellar. I've just been refactoring it bit by bit.